zebra-chain/src/lib.rs

```diff
@@ -6,7 +6,7 @@
 #![doc(html_favicon_url = "https://www.zfnd.org/images/zebra-favicon-128.png")]
 #![doc(html_logo_url = "https://www.zfnd.org/images/zebra-icon.png")]
 #![doc(html_root_url = "https://doc.zebra.zfnd.org/zebra_chain")]
-#![deny(missing_docs)]
+// #![deny(missing_docs)]
```

Comment:
Let's make sure this TODO is on a tracking issue somewhere?

Reply:
Went ahead and re enabled this in my follow up branch which should be the last PR in this chain